src/util/__tests__/composeUrl.test.ts

```diff
@@ -1,4 +1,4 @@
-import { composePath, composeUrl } from "./composeUrl";
```

Comment:
Why did you move everything inside a `__tests__` folder? 🤔

Reply:
No, really reason, mainly it was late and I was having a feeling it was leaking into the final builds. I am happy to revisit and move them back if you want.

Reply:
I usually prefer my tests aside the component (otherwise people forget about these 😬), but I get the leaking part, I have the same locally 😕 (I was playing a bit with tsdx today, to give a try and see if need all this extra configuration)